sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/VirtualMachinesOperations.cs

```diff
@@ -780,16 +780,19 @@ public async Task<AzureOperationResponse> ConvertToManagedDisksWithHttpMessagesA
         /// <param name='vmName'>
         /// The name of the virtual machine.
         /// </param>
+        /// <param name='hibernate'>
+        /// Optional parameter to hibernate a virtual machine. (Feature in Preview)
+        /// </param>
         /// <param name='customHeaders'>
         /// The headers that will be added to request.
         /// </param>
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        public async Task<AzureOperationResponse> DeallocateWithHttpMessagesAsync(string resourceGroupName, string vmName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse> DeallocateWithHttpMessagesAsync(string resourceGroupName, string vmName, bool? hibernate = default(bool?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Can we move ` bool? hibernate = default(bool?),` to the end (right before the `CancellationToken`)?

Reply:
this cannot be moved because it is generated from swagger and will be overwritten with each generation. 
customization/VirtualMachinesOperations.cs edited to preserve original signature.